examples/cifar10/cifar10-sim/README.md

```diff
@@ -115,12 +104,10 @@ If you have several GPUs available in your system, you can run simulations in pa
 
 ```
 export CUDA_VISIBLE_DEVICES=0
```

Comment:
just to confirm: current simulator does not support -t -gpu together in this manner

Reply:
maybe we could added a short sentence to note this

Reply:
Using this approach, each simulation run (including all clients) will run on one GPU.

Reply:
yeah, I mean if we say " -t 8 -n 8 -gpu 0 " supposedly it means run all 8 clients each with its own thread on gpu 0, but I guess it is not currently supported by simulator

Reply:
I see what you mean. I think one needs to provide one GPU index for each client. E.g., `-gpu 0,0,0,0,0,0,0,0`. Using CUDA_VISIBLE_DEVICES=0 is a shortcut for that. 
But I'll separate the two commands and add that they should be run in separate terminals.

Reply:
ah I see, sounds good, thanks!